src/utils/generation_utils.py

```diff
@@ -310,6 +315,7 @@ def do_generation(
     generation_for: str,  # For debugging output
     stop_tokens: Optional[List[str]] = None,
     new_file: bool = False,
+    streaming: bool = True,
```

Comment:
aside: this keeps everything in history, but doesn't stream the generation. I wonder if there will be things we don't want in history.

Reply:
The new_file param generates into a random new file other than chat_history.

Reply:
I guess I'm wondering if we will ever want ephemeral generation (append_output_to_file = False).